android/src/main/kotlin/com/kasem/media_picker_builder/MediaFile.kt

```diff
@@ -14,6 +14,7 @@ class MediaFile(
          * Only degrees 0, 90, 180, 270 will work.
          * */
         var orientation: Int,
+        var videoOrientation: Int,
```

Comment:
This seems confusing that there is already an `orientation` property on MediaFile on android and ios. Why not leverage what is already there?

Reply:
Was weird to me too, but for videos, we use it like for orientation of a thumbnail so I was forced to add another field

Reply:
Ok, I'll make sure it's one orientation, working for example and vid_camera app 🙃